### ISSUE TITLE
'UnicodeDecodeError' on windows powershell

### ISSUE THREAD

**Issue by zixuanweeei** [nvbn/thefuck#673]
Below are the warning and error when i execute command `fuck`

```
 RuntimeWarning: sys.stdin.encoding == 'utf-8', whereas sys.stdout.encoding == 'cp936', readline hook consumer may assume they are the same
  readline_hook.enable(use_pyreadline=use_pyreadline)
Traceback (most recent call last):
  File "d:\anaconda3\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "d:\anaconda3\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "D:\Anaconda3\Scripts\thefuck.exe\__main__.py", line 9, in <module>
  File "d:\anaconda3\lib\site-packages\thefuck\main.py", line 51, in main
    fix_command(known_args)
  File "d:\anaconda3\lib\site-packages\thefuck\main.py", line 27, in fix_command
    command = types.Command.from_raw_script(raw_command)
  File "d:\anaconda3\lib\site-packages\thefuck\types.py", line 125, in from_raw_script
    stderr = result.stderr.read().decode('utf-8')
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xb2 in position 6: invalid start byte
```

**Comment by zionyx** [comment]
I had the same issue as well.

**Comment by icylogic** [comment]
- Add an environment variable `PYTHONIOENCODING=utf-8` 
- OR change your $PROFILE from
  ```Powershell
  iex "$(thefuck --alias)"
  ```
  to
  ```Powershell
  $env:PYTHONIOENCODING="utf-8"
  iex "$(thefuck --alias)"
  ```

**Comment by zionyx** [comment]
Thanks, worked like a charm.

**Comment by ddspog** [comment]
That didn't work for me. I've set the environment variable, but no success.

I've even reinstalled thefuck with pip, with the environment variable already set. I was hoping that executing the whole python import thefuck with the correct encoding would solve the issue, but no.

So, waiting for more suggestions.

**Comment by kassick** [comment]
I was having this issue. I fixed it by adding a "chcp 65001"  to my $PROFILE

**Comment by frostming** [comment]
I am having the same issue with `PYTHONIOENCODING` set to `utf-8`.

I wonder if thefuck reads the stdout so any non-utf-8 chars will break this

**Comment by banjintaohua** [comment]
> I am having the same issue with `PYTHONIOENCODING` set to `utf-8`.
> 
> I wonder if thefuck reads the stdout so any non-utf-8 chars will break this.
> 
> PS: I am running thefuck on Python3.7 on a Chinese Windows system.
> 
> ### Solution:
> Add this line to profile:
> 
> ```powershell
> [Console]::OutputEncoding = [System.Text.Encoding]::UTF8
> ```

请问下, 怎么添加到 `profile` 呢?

**Comment by scorphus** [comment]
Fixed in c2df71caed917ea659a72a393b105881ec0709df.
